### DISCUSSION TITLE
immich - storage

### DISCUSSION THREAD

**Discussion by peeter22** [alexbelgium/hassio-addons#746] [Q&A] [answered]
Hi.
I am trying to configure the Immich addon for home assistant running on a virtual machine on a windows PC.
I would like to assign a folder on a sepparate drive to be used as a photo storage folder.
I have tried tinker with different combinations of localdisks, networkdisks and data_location but I was not able to make it work.

Even if in the logs I see that D: drive was mounted successfully, after uploading a photo I can not find it anywhere on the D drive or inside the home assistant file structure.

Can someone please point me to a direction how to set this up ?
![222976289-83adf96b-14bc-4b3d-a9e6-cf53018bbf9a](https://user-images.githubusercontent.com/57701560/222981185-b97c6f5b-61db-4b8f-9598-b1c0c0923f76.jpg)

**Comment by alexbelgium** [comment, marked as answer]
Hi, if you mount //ip/d, it will be mounted in /mnt/d. Therefore you need to put /mnt/d/photos in your data folder.

To have it mounted successfully, you need to fill the networkdisks (as you did) but also the cifs password and cifs username. It will be your username and password for the smb mount. It will be written in the log where the folder is mounted and if it is successful 

Good luck!

**Reply by peeter22** [reply]
Thanks a lot, you are awesome ! 
I have been strugling with this throughout the entire day.  I had no clue there even is a/mnt folder. I still have a lot to learn about HA.

**Reply by alexbelgium** [reply]
Thanks! Did it work? The /mnt folder is described in the log if the mount is successful

**Reply by peeter22** [reply]
yes, it worked like a charm. Thank you once again !
